src/component/mapping/new-mapping-destination.js

```diff
@@ -22,24 +22,19 @@ const NewMappingDestination = (props) => {
 
   let input;
   if (isLocal) {
-    input = {
-      className: 'input-file',
-      disabled: 'disabled',
-      placeholder: 'Choose file',
-      onClick: selectFile
-    };
+    input = <button  className="input-file btn waves-effect waves-light" onClick={selectFile}>Choose file</button>;
```

Comment:
Can we add `autofocus` to the button, like we do for the `input`? The user can hit `space` or `enter` to run the `onClick` of `<button>`, and it'd be convenient if, regardless of the type of destination, they can just start typing/hitting enter when they hit the page, and things will happen™

Also, you have one too many spaces between `button` and `className`, if you wanna change it while you're at it